guardian/bonobo
# Feature / count keys

## What does this change?

Shows stats on the front page about how many keys are active (i.e. unique keys used within a given time period)
This is calculated daily by KongKeyMon (https://github.com/guardian/kongkeymon/pull/14) and output into DynamoDB from where Bonobo picks it up

## How to test

- Run the KongKeyMon job as detailed on the above linked PR
- Deploy this to CODE and then load the frontpage. You should see a new box at the top of the screen (see image)

## How can we measure success?

Able to see key usage stats

## Have we considered potential risks?

n/a

## Images

<img width="615" alt="Screenshot 2022-08-31 at 16 53 24" src="https://user-images.githubusercontent.com/12482441/187729550-905afc62-67ea-481b-a20a-0e709697a9d5.png">
